lib/Constants.js

```diff
@@ -76,7 +76,8 @@ module.exports.VoiceOPCodes = {
     READY:               2,
     HEARTBEAT:           3,
     SESSION_DESCRIPTION: 4,
-    SPEAKING:            5
+    SPEAKING:            5,
+    DISCONNECT:          13
```

Comment:
Typings file line 207 needs updating for this plz

Reply:
Added